react-app/src/app/constants.js

```diff
@@ -1,18 +1,45 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
 import { faHome, faDiceOne, faUser, faSignOutAlt, faQuestion } from '@fortawesome/free-solid-svg-icons';
 
+import { removeState } from '../redux/localStorage';
+import { logout } from '../redux/Auth/actions';
+
+// eslint-disable-next-line
+const handleLogout = () => {
+  removeState('token');
+  removeState('tokenExpireDateTime');
+  // TODO: why does not work?
+  logout();
```

Comment:
You arent dispatching this action. It wont reach any reducer, since the logout function isnt getting any dispatch